.github/workflows/draft-new-release.yml

```diff
@@ -0,0 +1,77 @@
+name: "Draft new release"
+
+on:
+  issues:
+    types: [opened, labeled]
+
+jobs:
+  draft-new-release:
+    name: "Draft a new release"
+    runs-on: ubuntu-latest
+    # Only run for issues with a specific title and label. Not strictly required but makes finding the release issue again later easier.
+    # There is also a whitelist that you may want to use to restrict, who can trigger this workflow.
+    # Unfortunately, we cannot create an array on the fly, so the whitelist is just comma-separated.
+    if: startsWith(github.event.issue.title, 'Release version') && contains(github.event.issue.labels.*.name, 'release') && contains('thomaseizinger,bonomat,D4nte,da-kami,luckysori,tcharding,rishflab', github.event.issue.user.login)
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Extract version from issue title
+        run: |
+          TITLE="${{ github.event.issue.title }}"
+          VERSION=${TITLE#Release version }
```

Comment:
🤔 I wonder if we should do some regex validation on the version.

Reply:
Not sure I see the benefit. I tried to avoid as much shell as possible because it is the most brittle and hard to understand programming language on this planet :D

I also didn't see ant value in creating an action, just for this so I opted for bash.